server/src/index.ts

```diff
@@ -1,7 +1,11 @@
+import { config as setupEnv} from 'dotenv';
+// Setup environment variables
+setupEnv()
```

Comment:
Personal opinion: renaming exports is weird and confusing. You could just do this:


```js
import dotenv from 'dotenv';

dotenv.config()
```